hackathon_site/hardware/serializers.py

```diff
@@ -35,8 +35,9 @@ class Meta:
         fields = ("id", "name", "max_per_team", "unique_hardware_count")
 
     @staticmethod
-    def get_unique_hardware_count(obj: Category):
-        return Hardware.objects.filter(categories__id=obj.id).count()
+    def get_unique_hardware_count(obj: Category) -> int:
+        # return Hardware.objects.filter(categories__id=obj.id).count()
+        return obj.hardware_set.count()
```

Comment:
Nice change! This should be identical to what was there before, but add `distinct=True` to the `count` just in case (something we should have done before)